sentinelhub/io_utils.py

```diff
@@ -166,6 +165,13 @@ def read_numpy(filename: str) -> np.ndarray:
     return np.load(filename)
 
 
+def _create_parent_folder(filename: str) -> None:
+    """Create parent folder for input filename recursively"""
```

Comment:
what is recursive about this?

Reply:
`os.makedirs` is recursive directory creation function that makes all intermediate-level directories needed to contain the leaf directory 
docstring could be removed add almost zero meaning

Reply:
yes, in this case it's more confusing than explanatory.

Reply:
removed